subsys/bluetooth/mesh/Kconfig

```diff
@@ -513,6 +522,16 @@ config BT_MESH_RELAY_RETRANSMIT_INTERVAL
 	  messages, in milliseconds. Can be changed through runtime
 	  configuration.
 
+config BT_MESH_RELAY_SIMULTANEITY_COUNT
+	int "Maximum of simultaneous relay message support"
+	depends on BT_MESH_ADV_EXT_MULTI
+	depends on BT_EXT_ADV_MAX_ADV_SET > 1
+	default 1
```

Comment:
a range will default to the lowest value in the range if not specified.

So you can remove this.
```suggestion
```